apps/prairielearn/elements/pl-file-editor/pl-file-editor.py

```diff
@@ -20,20 +20,47 @@
 DIRECTORY_DEFAULT = "."
 NORMALIZE_TO_ASCII_DEFAULT = False
 
-
-def get_answer_name(file_name):
+def categorize_options(element: lxml.html.HtmlElement, data: pl.QuestionData) -> tuple[str, list, str]:
+    hasRange = "false"
```

Comment:
I think `hasRange` should be made into a bool and then the conversion to string should happen when it is loaded into the dictionary outside of this function (so the return type here should be `tuple[bool, list[list[int]], str]`